docusaurus/docusaurus.config.js

```diff
@@ -6,20 +6,21 @@ const darkCodeTheme = require('prism-react-renderer/themes/dracula');
 
 /** @type {import('@docusaurus/types').Config} */
 const config = {
-  title: 'My Site',
-  tagline: 'Dinosaurs are cool',
+  title: 'OSS Review Toolkit',
+  tagline: 'A suite of tools to assist with reviewing Open Source Software dependencies.',
```

Comment:
I recently changed this "tagline" a bit in the context of POM descriptions, and also the GitHub project page now says "A suite of tools to automate software compliance checks. ". I propose to use the same wording here.